### DISCUSSION TITLE
Make cost and consumption graphable values (without units) in Electricity_meter_Previous_Accumulative_Cost Charges array

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#340] [Ideas]
Was looking to make a simple graph of agile usage costs but apex won't show any data,  I think is it because the values are stored as text with the unit £ or kWh

Is it possible to make cost and consumption values more graphable (without units) in Electricity_meter_Previous_Accumulative_Cost Charges array?

**Comment by BottlecapDave** [comment]
Hi there. I've converted this to a [feature request](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/345). I'm not sure when I'll get to this, but happy to accept merge requests.
